### PR TITLE
Change associated consultations message depending on permissions AB#50806

### PR DESCRIPTION
**Pull request by RobOatesHistoricEngland**
Change associated consultations message depending on permissions

[AB#50806](https://hedev.visualstudio.com/76e71f30-b1ad-438a-a096-89ac98712f4e/_workitems/edit/50806)

### CODE REVIEW

File: arches_her/media/js/views/components/reports/scenes/resources.js [hunk 1]
```diff
@@ -94,6 +94,26 @@ define(['underscore', 'knockout', 'arches', 'utils/report', 'bindings/datatable'
...
+                        })
+                    } else {
+                        return false;
+                    }
+                }
+
+                if (!userCanViewConsultations()) {
+                    self.consultations_message = ko.observable('You do not have permission to view this data');
+                } else if (associatedConsultationsNode === undefined || associatedConsultationsNode.length === 0) {
+                    self.consultations_message = ko.observable('No consultations for this resource');
```
Comment (aj-he) [inline arches_her/media/js/views/components/reports/scenes/resources.js:115]: do you need to re-instantiate the `consultations_message` observable in lines 113 and 115 or can you just change the value, e.g. `self.consultations_message('You do not have permission to view this data');`??

**Review by aj-he — CHANGES_REQUESTED** [review]
see reinstatiation comment

**Review by csmith-he — APPROVED** [review]
Looks good to me

**Review by aj-he — APPROVED** [review]

**Review by Rehaan-Arif — COMMENTED** [review]
This has failed testing

**Comment by aj-he** [comment]
@RobOatesHistoricEngland and @csmith-he is this still current?

**Comment by csmith-he** [comment]
@aj-he @RobOatesHistoricEngland I think my work on the Consultations message superseded this.  Rob, am I correct in thinking that?

**Comment by RobOatesHistoricEngland** [comment]
@aj-he @csmith-he I also think that Chantelle's work superseded my PR.

**Comment by aj-he** [comment]
OK, I'll close this then

**Pull request closed without merging**